extensions/terminal-suggest/scripts/terminalScriptHelpers.ts

```
@@ -0,0 +1,51 @@
+import { platform } from 'os';
+import { exec } from 'child_process';
+import { promisify } from 'util';
+
+export const execAsync = promisify(exec);
+
+/**
+ * Cleans up text from terminal control sequences and formatting artifacts
+ */
+export function cleanupText(text: string): string {
+	// Remove ANSI escape codes
+	let cleanedText = text.replace(/\x1b\[\d+m/g, '');
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)